ddl/ddl_worker.go

```diff
@@ -779,6 +779,7 @@ func (w *worker) HandleJobDone(d *ddlCtx, job *model.Job, t *meta.Meta) error {
 	if err != nil {
 		return err
 	}
+	CleanupDDLReorgHandles(job, w.sessPool, w.concurrentDDL)
```

Comment:
CleanupDDLReorgHandles is enough? No need in HandleDDLJobTable?

Reply:
HandleDDLJobTable has a call to CleanupDDLReorgHandles as well, after the runDDLJob (which encapsulates the changes in a new transaction) is done. See line 881.